config.json.sample

```diff
@@ -7,5 +7,10 @@
     "url": "https://demo.owncloud.com/index.php/apps/oauth2/api/v1/token",
     "authUrl": "https://demo.owncloud.com/index.php/apps/oauth2/authorize"
   },
-  "apps" : ["files"]
-}
\ No newline at end of file
+  "apps" : ["files"],
+  "applications": [{
+	"name": "Files",
+	"iconMaterial": "folder",
```

Comment:
icons need to be provided by the app - they have to be loaded by url

Reply:
will be resolved in adjusted sample config